content/applications/sales/crm/track_leads/marketing_attribution.rst

```diff
@@ -0,0 +1,142 @@
+the results that fit the filter. For example, selecting the :guilabel:`Won` filter will only
+show won leads in the attribution report. :guilabel:`Group By`, found in the middle column, is
+used to organize the results into groups and can be used with or without filters.
+
+.. image:: marketing_attribution/search-results-multiple-options.png
+   :align: center
+   :alt: Select any number of filters and groups in the search options.
+
+.. tip::
+    Setting multiple Group By options will create nested groups according to which is selected
+    first. For example, selecting :guilabel:`Medium` followed by :guilabel:`Source` and then
+    ::guilabel:`Campaign` in :guilabel:`Group By` will sort all results *first* by medium, *then* by
+    the specific sources in each medium, followed by the campaigns in each source.
+
+    This can be verified by looking at the direction of the carat in the group tile in
+    the search bar.
+
+    .. image:: marketing_attribution/group-by.png
+       :align: center
+       :alt: The text in the tile is `Country > City`, showing that city is a subgroup of country.
+
+.. example::
+    For a useful first report:
+
+    #. Select the :guilabel:`Active` filter to view only leads that are still marked as active.
+    #. Select :guilabel:`Source` followed by the :guilabel:`City` or :guilabel:`Country`
+       groups depending on which grouping is more relevant.
+
+    .. image:: marketing_attribution/campaign-and-country-groups.png
+       :align: center
+       :alt: Each lead is now sorted by source, followed by city or country.
+
+    This report will contain all active leads grouped first by the source of the lead, then the
+    country or city each lead is from. This is useful to see the density of active opportunities
+    sorted by location.
+
+    With this data, marketing campaigns such as conferences or billboards can be targeted to the
+    locations generating the largest amount of potential revenue. Similarly, more attention can be
+    put toward increasing outreach in locations where existing marketing campaigns are less
+    effective.
+
+Export reports
+==============
+
+To set measures of the report, begin by navigating to the :icon:`oi-view-pivot`
+:guilabel:`(pivot view)`. Click the :guilabel:`Measures` button to view the available measures of
+the report. Select the desired measures, and verify the measures, filters, and groups are all
+displayed correctly in the pivot table.
+
+Next, click on the :icon:`fa-cog` :guilabel:`(settings)` button to
+the right of :guilabel:`Lead Analysis` in the top-left of the page to view the available export
+options.
+
+To export the report in the Odoo *Documents* app, navigate to :menuselection:`Settings app -->
+Spreadsheet --> Insert list in spreadsheet`. A menu will appear with export options.
+
+The report can be named using the `Name of the list` field. The number of items on the report can be
```

Comment:
```suggestion
The report can be named using the `Name of the link` field. The number of items on the report can be
```
If my suggestions aren't lining up with your intentions, just let me know. But this is what I'm seeing during my run-through...

![Screenshot 2024-07-15 094056](https://github.com/user-attachments/assets/3c8bd094-e165-4220-aa5d-89fc899635af)